assignments/closure.js

```diff
@@ -4,30 +4,76 @@
 // that manipulates variables defined in the outer scope.
 // The outer scope can be a parent function, or the top level of the script.
 
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+function even() {
+	return numbers.filter((num) => num % 2 === 0);
+}
+const closureTest = even();
+console.log('closureTest:', closureTest);
 
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 
 // ==== Challenge 2: Implement a "counter maker" function ====
 const counterMaker = () => {
```

Comment:
Maybe we could make the function parametric and use a while loop to automate the count process and make it more flexible.